rotkehlchen/chain/optimism/tokens.py

```diff
@@ -22,4 +24,9 @@ def _per_chain_token_exceptions(self) -> set[ChecksumEvmAddress]:
         Optimism ETH ERC20 token mirrors the user's balance on the chain.
         To avoid double counting, we exclude the token from the balance query.
         """
-        return {A_OPTIMISM_ETH.resolve_to_evm_token().evm_address, GIVPOW_ADDRESS}
+        return {
+            A_OPTIMISM_ETH.resolve_to_evm_token().evm_address,
+            GIVPOW_ADDRESS,
+            string_to_evm_address('0x0C6D90a98426bfD572a5c5Be572a7f6Bd1C5ED76'),  # Constant Inflow Token  # noqa: E501
+            string_to_evm_address('0xFb2b126660BE2fdEBa254b1F6e4348644E8482e7'),  # Constant Outflow Token  # noqa: E501
+        }
```

Comment:
seems that there is only one per chain of those https://docs.superfluid.org/docs/technical-reference/architecture#overview but I'm not sure 100%. Maybe you can dig further? A search on google points that there might be more

![image](https://github.com/user-attachments/assets/d6c99e6c-e883-4920-a1a3-ec804516559d)

but I didn't see even the op one listed on https://explorer.superfluid.org/optimism-mainnet/protocol . If you don't know we have a telegram chat and we can ping them

Reply:
> but I didn't see even the op one listed on https://explorer.superfluid.org/optimism-mainnet/protocol . If you don't know we have a telegram chat and we can ping them

thanks, the link was very helpful as you just need to select one of the tokens per chain and look for `CONSTANT_INFLOW_NFT` & `CONSTANT_OUTFLOW_NFT` methods